README.md

```diff
@@ -16,4 +16,8 @@ doesn't rely on this.
 
 
 ## Infrastructure
-Documentation on the infrastructure can be found [here](./docs/infrastructure-diagrams/README.md)
\ No newline at end of file
+Documentation on the infrastructure can be found [here](./docs/infrastructure-diagrams/README.md)
+
+## Dagit/Daemon 
+Code in the dagster folder [here](./dagster/README.md) is what is used to run the dagit and daemon interfaces
+on the platform. Code servers can then be hooked in and are part of different repos as needed.
```

Comment:
Not sure I understand this last sentence - could just be my limited knowledge of the project

Reply:
Fair point.  THe service works with three images communicating with each other. The dagit interface which handles the definitions and coordination, the daemon which handles the scheduling, and the code server which currently sits in the liia-tools-pipeline repo for the data platform, but could conceivably be easily reused for other projects by utilising a different code server.  This code is for the first two which are built using the Dockerfile here.

Reply:
Ah ok that makes sense - I think this explanation would fit well in this part of the readme (unless it already lives elsewhere)

Reply:
Good point. Those docs are in a different part of the repo in ./docs/ though I see that they're very out of date. Will need to update in a different PR.